src/layouts/Regulations.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <div class="w-1/2">
+        <RouterView></RouterView>
+    </div>
+</template>
+
+<script setup lang="ts">
```

Comment:
tüm sistemde script ler dosyanın üstünde  template altta

Reply:
o sıranın bir öenmi yok. Vue'da default olarak kendisi değiştiriyor bazen o sırayı.
Standardize etmek isçin değiştiririm

Reply:
Sıralamanın öneminin olmadığını biliyorum ama consistency önemli olduğunu da biliyorum.

 https://skiplist.com/insights/blog-software-principle-6-consistency-is-king  şuraya bakabilirsin. çok daha detaylı makalelerde var, en sade bulduğum buydu.